java/spacewalk-java.changes

```diff
@@ -1,3 +1,5 @@
+- Add endpoints to saltkeys API
```

Comment:
we had a discussion about enduser understandable changelogs. Maybe list the new endpoints ?

```suggestion
- Add new endpoints to saltkeys API: acceptedList, pendingList, rejectedList, deniedList, accept and reject
```